src/VisualStudio/Core/Def/Implementation/FindReferences/Contexts/WithoutReferencesFindUsagesContext.cs

```diff
@@ -88,6 +113,7 @@ protected override async ValueTask OnDefinitionFoundWorkerAsync(DefinitionItem d
                     {
                         EntriesWhenGroupingByDefinition = EntriesWhenGroupingByDefinition.AddRange(entries);
                         EntriesWhenNotGroupingByDefinition = EntriesWhenNotGroupingByDefinition.AddRange(entries);
+                        CurrentVersionNumber++;
```

Comment:
this looks to be a straight up bug that we were missing this.  we then became dependent on if we were called back into.  this would happen when we transitioned to the 'completed' state.  but is' still not ideal to not have this.